docs/commands/states.md

```diff
@@ -0,0 +1,300 @@
+---
+title: Movement Detections
```

Comment:
[Grammar] "Detection" should not be plural.

Suggested titles:
```suggestion
title: Movement Detection
```
```suggestion
title: Detecting Movement
```

Reply:
Using `states` for the files/folder was intentional to keep it short for the editors. While the title being a little more descriptive for the readers.
(Also just wanted to convey both when people shared links and such without extending the title itself)

The ` / ` was also intentional since beginner level readers find it easier to differentiate commands after copying.
> Each new line in a function file represents a new command. You may start a line with # to add comments. Commands in a function do not need to begin with a slash /, however doing so will not cause any errors.

It was consistent in all docs till someone edited out all the ` / ` in some docs a while ago.

But if you insist (believe it's still better) I can implement both of the suggested changes? 👀

"Detections" in the title is also intentional.
> The noun detection can be countable or uncountable. In more general, commonly used, contexts, the plural form will also be detection. However, in more specific contexts, the plural form can also be detections e.g. in reference to various types of detections or a collection of detections.